src/extension/app/store/bulk.js

```diff
@@ -653,6 +657,18 @@ export class BulkStore {
       rootEl.addEventListener('click', listener);
       rootEl.addEventListener('keyup', listener);
     }
+    // listen for url changes
+    const urlCheck = window.setInterval(() => {
+      if (this.appStore.sidekick.closest('body')) {
+        if (this.appStore.state !== STATE.FETCHING_STATUS
+          && this.appStore.location.href !== getLocation().href) {
+          this.selection = [];
+          this.appStore.fetchStatus(true, true);
+        }
+      } else {
+        window.clearInterval(urlCheck);
+      }
+    }, 500);
```

Comment:
I need to know the current `webPath` upfront so I can adjust the buttons based on the context. So I decided to bring the url change listener back. The upside is that we no longer have to fetch the status when the user starts a bulk operation.